report_thesis/src/sections/proposed_approach/testing_validation.tex

```diff
@@ -47,76 +61,142 @@ \subsection{Validation and Testing Procedures for Model Evaluation}\label{subsec
+By general representativeness, we mean ensuring that the test set reflects the general distribution of the dataset without being skewed by extreme values.
+This balance is essential for accurately assessing the model's performance on typical data points.
+
+Our method for handling extreme values ensures that the test set does not include samples outside the range seen in the training set.
+This approach addresses several critical challenges and represents a deliberate trade-off to improve the reliability of our model evaluation:
+
+Firstly, it mitigates the risk of uneven distribution of extreme values, which can disproportionately affect model performance metrics.
+If extreme values are unevenly distributed between the training and test sets, the evaluation of the model can be heavily skewed, leading to unreliable and misleading performance metrics.
+By redistributing extreme values evenly across the training folds, we ensure a more balanced and fair assessment of the model's capabilities during cross-validation.
+
+Secondly, this trade-off allows for a more stable and reliable assessment of the model's performance on typical data points.
+While the test set may be less representative of the full range of data, particularly regarding the rare extreme values, the evaluation focuses on the model's ability to generalize from the training data to new data within the same distribution range.
+This is prudent because, in many practical applications, the reliability of the model on typical data points is more critical than its performance on rare extremes.
+By using cross-validation in conjunction with a separate test set, we ensure that the model is robust and performs well under typical conditions.
+
+However, it is important to recognize the limitation this approach introduces: excluding extreme values from the test set means we can only confidently assess the model's performance within the range of the test set.
+If we receive predictions outside this range, we cannot reliably assert their accuracy, as the model has not been fully evaluated on such data.
+This could potentially make the model less useful in scenarios where predictions on extreme values are critical.
+Nevertheless, cross-validation partially mitigates this issue by allowing the model to learn from extreme values during the training and validation phases, thereby improving its robustness.
+Metrics obtained from cross-validation provide a broader picture of the model's accuracy and robustness across the entire dataset, including extreme values.
+
+Therefore, although our approach may make the test set less representative of the full dataset, it is a deliberate trade-off aimed at achieving a more accurate and reliable evaluation of the model's generalization performance.
```

Comment:
Added a discussion of the tradeoffs for our method, which I thought it was very important to highlight. It isn't perfect, but it's the most-ideal we've found. Need to highlight that. 

Please read this thoroughly! Would really like to hear your feedback on this. Does it make sense? Do the claims follow from our reasoning?

Reply:
Very good overall IMO, some very slight/minor phrasing issues which I have attempted to correct through my suggestions.